src/main/java/uk/gov/digital/ho/hocs/casework/api/BankHolidayResource.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.digital.ho.hocs.casework.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.digital.ho.hocs.casework.domain.repository.BankHolidayRepository;
+
+@RestController
+public class BankHolidayResource {
+    BankHolidayService bankHolidayService;
+    BankHolidayRepository bankHolidayRepository;
+
+    @Autowired
+    public BankHolidayResource(BankHolidayService bankHolidayService) {
+        this.bankHolidayService = bankHolidayService;
+    }
+
+    @GetMapping(value = "/bankHoliday/refresh")
+    public ResponseEntity<String> refreshFromApi() {
```

Comment:
I was thinking about using either Quartz or Shedlock to have this happen automatically, but thinking about it, how often is it going to run? twice a year? it probably isn't worth adding either of those libraries to the project just for this feature, but if we add it for something else it might be worth moving this over to use that.